packages/specifications/components/form-field/specifications-form-field.mdx

```diff
@@ -0,0 +1,13 @@
+import {Description} from '@storybook/addon-docs';
+
+[//]: # (import Specs from '@ovhcloud/ods-core/src/components/form-field/docs/spec.md';)
+import SpecsFormFieldContents from './specifications-form-field-contents.mdx';
+import SpecsFormFieldTests from './specifications-form-field-tests.mdx';
+
+[//]: # (<Description>{Specs}</Description>)
```

Comment:
Same here: [//]: # (<Description>{Specs}</Description>) to <Description>{Specs}</Description>